src/Hooks/useLocalStorage.jsx

```diff
@@ -0,0 +1,24 @@
+import React, { useState } from 'react';
+import getToken from '../lib/tokens';
+
+export const useLocalStorage = (key, initialValue) => {
```

Comment:
👍 I really like this abstraction, nice work 👏.

Reply:
Great job!

Reply:
Thanks for checking it.